pkg/operator/operator_test.go

```diff
@@ -665,3 +665,15 @@ func sanitizeEBSCSIDriver(instance *v1alpha1.EBSCSIDriver) {
 		return instance.Status.Conditions[i].Type < instance.Status.Conditions[j].Type
 	})
 }
+
+func defaultImages() images {
+	return images{
+		csiDriver:           "registry.svc.ci.openshift.org/ocp/4.5:aws-ebs-csi-driver",
+		provisioner:         "registry.svc.ci.openshift.org/ocp/4.5:csi-external-provisioner",
+		attacher:            "registry.svc.ci.openshift.org/ocp/4.5:csi-external-attacher",
+		resizer:             "registry.svc.ci.openshift.org/ocp/4.5:csi-external-resizer",
+		snapshottter:        "registry.svc.ci.openshift.org/ocp/4.5:csi-external-snapshotter",
+		nodeDriverRegistrar: "registry.svc.ci.openshift.org/ocp/4.5:csi-node-driver-registrar",
+		livenessProbe:       "registry.svc.ci.openshift.org/ocp/4.5:csi-liveness-probe",
+	}
```

Comment:
shouldn't the sidecar just default to `origin-csi-external-attacher` etc ?

Reply:
This test isn't pulling those images, so I think it's OK.

The problem in using `latest` is that we want a test running in an OCP 4.4 cluster to pull the `4.4` sidecars.

Reply:
fixed the sidecars, still the CSI driver itself is not mirrored to quay.io yet